src/web/app/components/comment-box/comment-row/comment-row.component.spec.ts

```diff
@@ -45,4 +46,114 @@ describe('CommentRowComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
```

Comment:
the expectations in some of the tests are incorrect, let's create spys:

```javascript
  const spyVisibilityStateMachine: any = {
    allowAllApplicableTypesToSee: jest.fn(),
    applyVisibilitySettings: jest.fn(),
    getVisibilityTypesUnderVisibilityControl: jest.fn()
  }

  const spyCommentService: any = {
    getNewVisibilityStateMachine: jest.fn().mockReturnValue(spyVisibilityStateMachine)
  };
```

and provide them in configureTestingModule: 

```javascript
providers: [
  { provide: FeedbackResponseCommentService, useValue: spyCommentService },
]
```